packages/integrations/node/src/serve-static.ts

```diff
@@ -0,0 +1,79 @@
+import path from "node:path";
+import url from "node:url";
+import send from "send";
+import type { IncomingMessage, ServerResponse } from "node:http";
+import type { Options } from "./types.js";
+import type { NodeApp } from "astro/app/node";
+
+/**
+ * Intended to be used only in the standalone mode.
+ * This is queried first, and if a static file is not found, it relegates to the SSR handler.
+ */
+export default function createListener(app: NodeApp, options: Options) {
+	const client = resolveClientDir(options);
+    return (req: IncomingMessage, res: ServerResponse, ssr: () => unknown) => {
```

Comment:
Can we add a small comment that explains what the third argument is?

Reply:
Added

https://github.com/withastro/astro/pull/9582/files#diff-867b6a94d71f10420472afd23eedb1ed912505336f406d23ac93e502e0810ad4R17